src/dialog.c

```diff
@@ -275,8 +292,6 @@ void dialog_draw(Dialog *dialog) {
 		return;
 	}
 
-	float o = dialog->opacity;
```

Comment:
Not sure why this was changed. If it's for readability purposes, then consider re-introducing the variable and renaming it `opacity`. I can see how `o` may be unclear.

It is often the case in drawing code that alpha/opacity-style state variables are filtered through a function rather than consumed directly (often using one of the [easing functions](https://github.com/taisei-project/cglm/blob/ee15b56bd90941c9c27dd409df4802490645f367/include/cglm/ease.h)). While this doesn't apply here, I think it's nice to cache stuff like this into local variables by convention, to facilitate such changes.